lib/routes/anthropic/news.ts

```diff
@@ -0,0 +1,60 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['programming'],
+    example: '/anthropic/news',
+    parameters: {},
+    radar: [
+        {
+            source: ['anthropic.com'],
+        },
+    ],
+    name: 'Anthropic News',
```

Comment:
```suggestion
    name: 'News',
```
No need to repeat namespace name https://github.com/DIYgod/RSSHub/blob/3223e40ee963d43dedcdc04e7a3cbf103405d320/lib/routes/anthropic/namespace.ts#L4 in route name